src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -136,7 +136,8 @@ public List<Car> getAllByDriver(Long driverId) {
                 cars.add(getCar(resultSet));
             }
         } catch (SQLException e) {
-            throw new DataProcessingException("Can't get list cars for car with id " + driverId, e);
+            throw new DataProcessingException("Can't get list cars on this driverId "

```

Comment:
throw new DataProcessingException("Can't get list of cars from driver ID: " + driverId, e);